test/README.md

```diff
@@ -19,3 +19,23 @@ To install `tmt` on fedora at least those packages are needed:
 ```shell
 sudo dnf install tmt tmt+provision-virtual
 ```
+
+Ansible container
+-----------------
+To execute real tests against the VM ansible is used. Ansible is run from within a container.
+
+⚠️ When changing `execution-environment.yml` you need to rebuild and commit `ansible-container/` as described below
+
+```shell
+ansible-builder create --context ansible-container
```

Comment:
Can we add a job that runs this in a github runner and fails if there are changes?  We do similar things with gofmt.